### ISSUE TITLE
Array of structs, containing an array not parsed properly

### ISSUE THREAD

**Issue by MiMaKes** [sergiolindau/mathjslab#1]
Assigning an array to an element of a struct array is not working as expected. This can be reproduced with the following example:

```MATLAB
StructArray(1).Name = 'Array1';
StructArray(1).DataArray = [1, 2, 3, 4, 5];

StructArray(2).Name = 'Array2';
StructArray(2).DataArray = [6, 7, 8, 9, 10];
```

Gives following error:
![image](https://github.com/user-attachments/assets/d76376ec-c3ba-4c81-9133-84348b43491e)

**Comment by sergiolindau** [comment]
Yes, Structure no yet full implemented. I needed to comment out the basic test definition in the `src/Structure.spec.ts` file. I'll check that out.
